platform/email/src/main/java/org/motechproject/email/service/impl/EmailSenderServiceImpl.java

```diff
@@ -42,6 +43,7 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(EmailSenderServiceImpl.class);
 
     @Override
+    @Transactional(noRollbackFor = MailException.class)
```

Comment:
Will this work when we are participating in an existing transaction? Regardless, I would make MailException a checked exception in this case.